allennlp/commands/find_learning_rate.py

```diff
@@ -106,15 +122,16 @@ def find_learning_rate_from_args(args: argparse.Namespace) -> None:
     params = Params.from_file(args.param_path, args.overrides)
     find_learning_rate_model(params, args.serialization_dir,
                              args.start_lr, args.end_lr,
-                             args.num_batches, args.linear, args.stopping_factor)
+                             args.num_batches, args.linear, args.stopping_factor, args.force)
 
 def find_learning_rate_model(params: Params,
                              serialization_dir: str,
                              start_lr: float,
                              end_lr: float,
                              num_batches: int,
                              linear_steps: bool,
-                             stopping_factor: Optional[float]) -> None:
+                             stopping_factor: Optional[float],
+                             force: bool) -> None:
```

Comment:
This should probably default to `False`, for backwards compatibility (and this is why the tests are failing).

Reply:
Oops, just fixed that and added a test with `force=True`